### PR TITLE
Include additional redis keyword arguments

### PR DESCRIPTION
**Pull request by untari**
This pull request is to add redis keywords arguments, and it has been tested in the locally.

### CODE REVIEW

**Comment by hongquan** [comment]
Which page to test this? I can only found `/control/` but it doesn't have WebSocket feature, whereas, by looking into settings.py, the `redis_connection_kwargs` is used for channel layer config, which belongs to WebSocket feature.

File: server/venueless/settings.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 from django.utils.crypto import get_random_string
 from kombu import Queue
+from redis.asyncio.retry import Retry
```
Comment (hongquan) [inline server/venueless/settings.py:8]: The Redis client used in channel layer is non-async one, you should use `Retry` from non-async module also.

**Review by hongquan — CHANGES_REQUESTED** [review]

File: server/venueless/settings.py (hunk 1 above)
Comment (hongquan) [reply server/venueless/settings.py:8]: Ah no, my mistake.

**Review by hongquan — APPROVED** [review]

**Comment by mariobehling** [comment]
Thanks!

**Pull request merged**